frontend/src/metabase/static-viz/components/RowChart/RowChart.tsx

```diff
@@ -51,16 +51,19 @@ interface StaticRowChartProps {
   getColor: ColorGetter;
 }
 
-const staticTextMeasurer: TextWidthMeasurer = (
-  text: string,
-  style: FontStyle,
-) =>
-  measureText(
+const staticTextMeasurer: TextMeasurer = (text: string, style: FontStyle) => {
+  const textWidth = measureText(
     text,
     parseInt(style.size.toString(), 10),
     style.weight ? parseInt(style.weight.toString(), 10) : 400,
   );
 
+  return {
+    width: textWidth,
+    height: -1, // this will be ignored by RowChart
```

Comment:
This is a bit confusing. If `height` is not used, then previous interface made more sense.

How about we apply [this suggestion](https://github.com/metabase/metabase/pull/32057/files#r1250027757) but improve the naming, i.e. `measureTextWidth: TextWidthMeasurer` + `measureTextDimensions: TextDimensionsMeasurer`? 

`measureText` would be no more.